package.json

```diff
@@ -14,12 +14,14 @@
     "lint:fix": "./node_modules/.bin/eslint . --fix"
   },
   "dependencies": {
-    "bunyan": "^1.8.9",
-    "shr-models": "^5.2.1"
+    "bunyan": "^1.8.9"
   },
   "devDependencies": {
     "chai": "^3.5.0",
```

Comment:
bunyan and chai are used like peer dependencies. I think that we haven't hit any errors because new versions of those libraries haven't really been released, however if shr-test-helpers' bunyan or chai became desynced from a parent module we could probably run into issues.

Reply:
OK.  I buy that argument.  Neither have caused any problems thus far, but it probably makes sense.